src/app/components/overview-stats/overview-stats.component.ts

```
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-overview-stats',
+  templateUrl: './overview-stats.component.html',
+  styleUrls: ['./overview-stats.component.scss']
+})
+export class OverviewStatsComponent implements OnInit {
+
+  constructor() { }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)